services/app-api/action/changeStatusAny.js

```diff
@@ -43,7 +44,7 @@ export const changeStatusAny = async (event, config) => {
   try {
     const theEmails = await Promise.all(
       config.emailFunctions.map(
-        async (f) => await f(updatedPackageData, config)
+        async (f) => await f(updatedPackageData, config, user)
```

Comment:
This is a fine way to do it, but the user details are also in the updatedPackageData as lastModifiedName and lastModifiedEmail.